app/controllers/api/functions.php

```diff
@@ -1494,6 +1557,59 @@
         $response->noContent();
     });
 
+App::patch('/v1/functions/:functionId/deployments/:deploymentId/builds/:buildId')
```

Comment:
Considering deploymentId and buildId is oneToOne relationship, I don't like you need to specify both on endpoint. We could get buildId using subqueries.

Let's discuss with Christy or Eldad how this endpoint could look like, and how we can rename existing one (redeploy endpoint)

Reply:
Updating the endpoint to `deployments/:deploymentId/build` as discussed